server/metrics.py

```diff
@@ -113,6 +135,12 @@
     ["game_mode", "game_state"],
 )
 
+rated_games = Counter(
+    "server_game_rated_games_total",
+    "Number of rated games",
+    ["leaderboard"]
+)
+
```

Comment:
I wonder if this would be better as a label on the `active_games` metric

Reply:
There is only really the faf gamemode that has more than one leaderboard. The documentation advises against having too many possible combinations of lables as that is bad for performance. I would want to have these two in different graphs anyway, so I think it is better to keep them separate.